zipline/finance/commission.py

```diff
@@ -59,7 +134,57 @@ def calculate(self, order, transaction):
         raise NotImplementedError('calculate')
 
 
-class PerShare(CommissionModel):
+class EquityCommissionModel(CommissionModel):
```

Comment:
I updated the class hierarchies to be simpler, we now use an abstractproperty called `allowed_asset_types` to define what is supported.

> A short docstring on these classes would be nice (e.g. for sphinx docs).

Will add